common/source/docs/common-siyi-zr10-gimbal.rst

```diff
@@ -81,7 +81,20 @@ Ethernet Connectivity
 
 Instead of using a UART serial connection to control the gimbal, an Ethernet virtual serial connection may also be used, as well as transmitting a video stream from the camera.
 
-An example of the setup of the gimbal in a networked ArduPilot vehicle system is detailed in :ref:`common-ethernet-vehicle`
+An example of the setup of the gimbal in a networked ArduPilot vehicle system is detailed in :ref:`common-ethernet-vehicle` but in short, to control the gimbal over Ethernet
```

Comment:
```suggestion
An example of the setup of the gimbal in a networked ArduPilot vehicle system is detailed in :ref:`common-ethernet-vehicle` but in short, to control the gimbal over Ethernet:
```

Reply:
such a nit, I am merging anyway

Reply:
Thanks, I actually debated with myself for a moment whether to put a colon on the end or not.  :-)